### PR TITLE
feat: allow pull_request_creator to create draft PRs

### PR DESCRIPTION
**Pull request by rgraber**

### CODE REVIEW

File: jenkins/pull_request_creator.py [hunk 1]
```diff
@@ -172,11 +174,14 @@ def create(self, delete_old_pull_requests):
     default=True,
     help="If set, delete old branches with the same base branch name and close their PRs"
 )
+@click.option(
+    '--draft', is_flag=True
```
Comment (robrap) [inline jenkins/pull_request_creator.py:178]: I assume `False` is the default default?

**Review by robrap — APPROVED** [review]
Looks good, as long as it works. :) 

Note: I think if you update https://github.com/edx/testeng-ci/blob/master/pylintrc you'll get rid of the fstring errors.

File: jenkins/pull_request_creator.py (hunk 1 above)
Comment (rgraber) [reply jenkins/pull_request_creator.py:178]: If it's specified with is_flag, then yes

**Review by adzuci — COMMENTED** [review]
Thanks for adding this!

**Pull request merged**